app/javascript/VideoEditorComponent.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import VideoEditorComponent from "./VideoEditorComponent";
+
+describe("VideoEditorComponent", () => {
+  describe("#toPreview", () => {
+    const tests = [
+      {
+        alt: "alt text",
+        url: "https://www.youtube.com/watch?v=LO2k-BNySLI",
+        expected: (
+          <div className="video">
+            <iframe
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowFullScreen
+              frameBorder="0"
+              height="315"
+              src="https://www.youtube-nocookie.com/embed/LO2k-BNySLI"
+              title="alt text"
+              width="560"
+            />
+          </div>
+        ),
+      },
+      {
+        url: "https://www.youtube.com/watch?v=X_eXSzyZudM&somebogusarg=foo",
+        expected: (
+          <div className="video">
+            {/* eslint-disable-next-line jsx-a11y/iframe-has-title */}
```

Comment:
[question] should we just be making the alt-text required, rather than being able to output something that needs the eslint rule disabled?

Reply:
Yes, I think that would be a cleaner and more accessible solution. I will make the change